__test__/api/comments/createSingleComment.test.js

```diff
@@ -0,0 +1,75 @@
+const app = require("../../../server");
+const supertest = require("supertest");
+const CommentModel = require("../../../models/comments");
+const request = supertest(app);
+
+describe("POST /comments", () => {
+  let comment;
+
```

Comment:
Hey, where is the beforeEach function where you create the comment that you delete in the afterEach function.

Reply:
since the endpoint creates the entry in db, only `afterEach` was required to cleanup db